Index: modules/generators/mod_autoindex.c
===================================================================
--- modules/generators/mod_autoindex.c	(revision 1855386)
+++ modules/generators/mod_autoindex.c	(working copy)
@@ -1911,6 +1911,8 @@
     struct ent *c2;
     int result = 0;
 
+    return 0;
+
     /*
      * First, see if either of the entries is for the parent directory.
      * If so, that *always* sorts lower than anything else.
Index: server/main.c
===================================================================
--- server/main.c	(revision 1855386)
+++ server/main.c	(working copy)
@@ -463,7 +463,11 @@
     destroy_and_exit_process(process, 1);
 }
 
-int main(int argc, const char * const argv[])
+#ifdef HFND_FUZZING_ENTRY_FUNCTION
+ HFND_FUZZING_ENTRY_FUNCTION(int argc, const char *const *argv)
+#else
+ int main(int argc, const char * const argv[])
+#endif
 {
     char c;
     int showcompile = 0, showdirectives = 0;
Index: server/request.c
===================================================================
--- server/request.c	(revision 1855386)
+++ server/request.c	(working copy)
@@ -1389,7 +1389,7 @@
     return OK;
 }
 
-
+__attribute__((no_sanitize("memory")))
 AP_DECLARE(int) ap_location_walk(request_rec *r)
 {
     ap_conf_vector_t *now_merged = NULL;
Index: server/util_pcre.c
===================================================================
--- server/util_pcre.c	(revision 1855386)
+++ server/util_pcre.c	(working copy)
@@ -301,6 +301,7 @@
     }
 }
 
+__attribute__((no_sanitize("memory")))
 AP_DECLARE(int) ap_regname(const ap_regex_t *preg,
                            apr_array_header_t *names, const char *prefix,
                            int upper)
